data/beta/field/local-authority-eng.yaml

```diff
@@ -1,4 +1,4 @@
-text: "A local authority in England."
+text: "The local authority's ISO 3166-1 alpha 2 code. Subsidiary authorities are given a Standard Names and Codes (SNAC) code instead."
```

Comment:
The SNAC part is incorrect, and we cannot say some authorities are "subsidiary".